recipe/meta.yaml

```diff
@@ -1,52 +1,54 @@
-{% set version = "2.1.0" %}
+{% set version = "2.3.0" %}
 
 package:
   name: pyviz_comms
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/p/pyviz_comms/pyviz_comms-{{ version }}.tar.gz
-  sha256: f4a7126f318fb6b964fef3f92fa55bc46b9218f62a8464a8b18e968b3087dbc0
+  sha256: 44709b2e639e84eb1ae7be5eaf3e28a9be7932d29a8cefce1cd603cb3904ad9c
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  skip: true  # [py<37]
+  script:
+    - {{ PYTHON }} -m pip install . --no-deps -vv --no-build-isolation
 
 requirements:
   host:
-    - python >=2.7
+    - python
     - param
     - pip
-    - jupyter-packaging >=0.7.9
-    - jupyterlab >=3.0
-    - notebook
-    - setuptools
-    - twine
```

Comment:
Isn't twine, rfc3986 and keyring required on build here https://github.com/holoviz/pyviz_comms/blob/abc8f4dfc06c3981aeb1749af91b506dd3e095fc/setup.py#LL74C1-L74C1?

Reply:
These were introduced because for a brief period our CI was complaining that they were not pulled in. I think they are actually only needed to upload the packages to PyPI with twine.

Reply:
i.e. `twine` is only used for PyPI uploads and for a while `twine` was not correctly expressing it's dependencies on `rfc3986` and `keyring` so we added those manually. These can all safely be dropped.

Reply:
kk